### PR TITLE
K8SPSMDB-558: ignore annotations changes

### PR DESCRIPTION
**Pull request by defbin**
[K8SPSMDB-558](https://jira.percona.com/browse/K8SPSMDB-558) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/794/commits/108ee6643ce9ec6754f79ecc1fe4f9a0ce37c9e1
image: `perconalab/percona-server-mongodb-operator:PR-794-108ee664`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| one-pod | failed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| demand-backup | failed |
| scaling | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| rs-shard-migration | passed |
| monitoring-2-0 | passed |
| scheduled-backup | passed |
| arbiter | passed |
| demand-backup-sharded | failed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | failed |
| data-sharded | passed |
| non-voting | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/794/commits/395ab12f163214be9c82caad47766ca35e3465b4
image: `perconalab/percona-server-mongodb-operator:PR-794-395ab12f`

**Review by egegunes — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | failed |
| storage | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| demand-backup | passed |
| scaling | passed |
| scheduled-backup | failed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| rs-shard-migration | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| demand-backup-sharded | failed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | failed |
| upgrade | failed |
| data-sharded | passed |
| non-voting | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/794/commits/9ea6846b1b18fe336f648cc60afe9b900fdac145
image: `perconalab/percona-server-mongodb-operator:PR-794-9ea6846b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| one-pod | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| scheduled-backup | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| rs-shard-migration | passed |
| demand-backup-sharded | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| upgrade | passed |
| users | passed |
| upgrade-sharded | passed |
| data-sharded | passed |
| pitr | passed |
| non-voting | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/794/commits/68e9fbb053a0db887a350d7a0318c188528bb3e1
image: `perconalab/percona-server-mongodb-operator:PR-794-68e9fbb0`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| data-sharded | passed |
| non-voting | passed |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| self-healing | passed |
| scaling | passed |
| self-healing-chaos | passed |
| security-context | passed |
| operator-self-healing | passed |
| rs-shard-migration | passed |
| operator-self-healing-chaos | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/794/commits/68e9fbb053a0db887a350d7a0318c188528bb3e1
image: `perconalab/percona-server-mongodb-operator:PR-794-68e9fbb0`

File: Jenkinsfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ void CreateCluster(String CLUSTER_PREFIX) {
             gcloud auth activate-service-account --key-file $CLIENT_SECRET_FILE
             gcloud config set project $GCP_PROJECT
             gcloud container clusters list --filter $CLUSTER_NAME-${CLUSTER_PREFIX} --zone $GKERegion --format='csv[no-heading](name)' | xargs gcloud container clusters delete --zone $GKERegion --quiet || true
-            gcloud container clusters create --zone $GKERegion $CLUSTER_NAME-${CLUSTER_PREFIX} --cluster-version=1.21 --machine-type=n1-standard-4 --preemptible --num-nodes=3 --network=jenkins-vpc --subnetwork=jenkins-${CLUSTER_PREFIX} --no-enable-autoupgrade
+            gcloud container clusters create --zone $GKERegion $CLUSTER_NAME-${CLUSTER_PREFIX} --cluster-version=1.20 --machine-type=n1-standard-4 --preemptible --num-nodes=3 --network=jenkins-vpc --subnetwork=jenkins-${CLUSTER_PREFIX} --no-enable-autoupgrade
```
Comment (delgod) [inline Jenkinsfile]: why we rolled back version here?

**Review by delgod — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (defbin) [reply Jenkinsfile]: this is old commit. right now here the gke version is the same as in main branch. also pxco also use 1.20.
I don't mind which version to use. but would like to see the same gke version for both operators

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| monitoring-2-0 | passed |
| limits | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| demand-backup-sharded | failed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| data-sharded | passed |
| pitr | passed |
| non-voting | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/794/commits/5a0196c0ed8ef0b326dd8d96c6fe8e950c1b7d12
image: `perconalab/percona-server-mongodb-operator:PR-794-5a0196c0`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| storage | passed |
| users | passed |
| self-healing | passed |
| data-sharded | passed |
| self-healing-chaos | passed |
| non-voting | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| security-context | passed |
| rs-shard-migration | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/794/commits/5a0196c0ed8ef0b326dd8d96c6fe8e950c1b7d12
image: `perconalab/percona-server-mongodb-operator:PR-794-5a0196c0`

**Pull request merged**